house/tests/house_login_page_tests.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from house.constants import LOGIN_PAGE_ROUTE
+from house.views import house_login
+from house.models import House
+from factories.house import HouseFactory
+from django.test.client import RequestFactory
+from house.forms import HouseIDForm
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+
+
+# Fixures:
+
+@pytest.fixture
+def request_factory():
+    return RequestFactory()
+
+
+@pytest.fixture
+def generate_get_request(request_factory):
+    get_request = request_factory.get('login/')
+    return get_request
+
+
+@pytest.fixture
+def generate_form(request_factory):
+    HouseFactory().save()
+    house = House.objects.all()[0]
+    post_request = request_factory.post('login/', {'house_id': f'{house.house_id}'})
+    form = HouseIDForm(post_request.POST)
+    return form
+
+
+@pytest.fixture
+def generate_bad_form(request_factory):
+    post_request = request_factory.post('login/', {'house_id': '1234'})
+    form = HouseIDForm(post_request.POST)
+    return form
+
+
+# Tests:
```

Comment:
Not needed

Reply:
"No comments for me please" Rony Yosef
:)
I'll take it off